common.blocks/history/_provider/history_provider_history-api.js

```diff
@@ -54,10 +54,18 @@ provide(inherit(Base, /** @lends history.prototype */{
     },
 
     _syncState : function() {
-        // Replace null with undefined to catch initial popstate
-        if(window.history.state === null) {
-            window.history.replaceState(undefined, document.title, window.location.href);
+        var state = window.history.state;
+
+        // Replace null with undefined to catch initial popstate.
+        // Initial val must be null (in specification).
+        // But in the UC browser for Android history.state equals null.
+        if (state === null || typeof state === 'undefined') {
```

Comment:
please run linter before commit (see https://travis-ci.org/bem/bem-history/jobs/117659145 for details).